src/main/java/com/rcv/GuiConfigController.java

```diff
@@ -159,9 +178,8 @@ public void buttonOutputDirectoryClicked() {
     dc.setTitle("Output Directory");
 
     File outputDirectory = dc.showDialog(null);
-    if (outputDirectory != null) {
```

Comment:
I'd prefer to stick with braces even for single-line cases like this, if you don't mind. We've been doing that everywhere else.

Reply:
I don't think I've been following that convention in my code, but I'll put these ones back in in my next commit.